.bazelci/presubmit.yml

```diff
@@ -44,15 +38,20 @@ platforms:
     - "-//images/gcloud-bazel:gcloud-layer"
     build_flags:
     - "--verbose_failures"
-    # add the python2 symlink to the PATH
-    # see https://github.com/bazelbuild/continuous-integration/issues/854
-    # TODO(nlopezgi): remove once issue above is fixed
-    - "--action_env=PATH=/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/tmp/bin/"
     test_targets:
     - "..."
     test_flags:
     - "--verbose_failures"
-    # add the python2 symlink to the PATH
-    # see https://github.com/bazelbuild/continuous-integration/issues/854
-    # TODO(nlopezgi): remove once issue above is fixed
-    - "--action_env=PATH=/usr/local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/tmp/bin/"
+  min_supported_version:
+    name: "Minimum Supported Version"
+    bazel: "2.2.0"
+    platform: ubuntu1604
+    build_targets:
+    - "--"
+    - "..."
+    - "-//images/gcloud-bazel:gcloud_push"
```

Comment:
I forget, why were these disabled? Seems like they should be enabled

Reply:
Probably didn't build at the time or something, let's see if they work now.